server/src/main/java/org/opensearch/index/store/RemoteSegmentStoreDirectory.java

```diff
@@ -859,6 +919,10 @@ public void deleteStaleSegments(int lastNMetadataFilesToKeep) throws IOException
         logger.debug("deletedSegmentFiles={}", deletedSegmentFiles);
     }
 
+    public Map<Long, String> getMetadataFilePinnedTimestampMap() {
```

Comment:
Not used any where in code. we can remove for now we can need if need in follow up PR's.